platforms/android/xwalk.gradle

```diff
@@ -29,7 +29,18 @@ def DEFAULT_MIN_SDK_VERSION = 14
 
 def getConfigPreference(name) {
     name = name.toLowerCase()
-    def xml = file("res/xml/config.xml").getText()
+
+    def xml
+
+    // cordova-android >= 7.0.0
+    def configFile = file("src/main/res/xml/config.xml")
```

Comment:
I can see you're no longer using `xml` for multiple purposes. However, I still believe the style here doesn't look as "even" as it could (`configFile` is used in only one of the branches, for example). My suggestion of iterating through the possible location remains.

Reply:
Feel free:

![2018-01-15-091544_162x50_escrotum](https://user-images.githubusercontent.com/12952835/34932800-b28fbd34-f9d4-11e7-85a4-8f589218d99b.png)